DuckDuckGo/Preferences/View/PreferencesVPNView.swift

```diff
@@ -112,6 +113,34 @@ extension Preferences {
                 }
                 .padding(.bottom, 12)
 
+                if model.exclusionsAvailable && model.appExclusionsFeatureEnabled {
```

Comment:
If exclusions are available to the current platform, and app exclusions are enabled... show new UI.